docs/tables/aws_wellarchitected_lens_review.md

```diff
@@ -0,0 +1,62 @@
+# Table: aws_wellarchitected_lens_review
+
+Review the state of your applications and workloads against architectural best practices, identify opportunities for improvement, and track progress over time.
+
+## Examples
+
+## Basic info
+
+```sql
+select
+  lens_name,
+  workload_id,
+  lens_arn,
+  lens_alias,
+  lens_version,
+  updated_at
+from
+  aws_wellarchitected_lens_review;
+```
+
+## List reviews for deprecated lenses
```

Comment:
```suggestion
### List reviews of deprecated lenses
```

Reply:
@ParthaI I think you need to add the `pillar_review_summaries` and the `risk_counts` columns to the query since those fields reflect the lens's review.